linker/Linker/Annotations.cs

```diff
@@ -423,5 +434,60 @@ public void SaveDependencies ()
 			zipStream = null;
 			dependency_stack = null;
 		}
+
+		bool WillAssemblyBeModified (AssemblyDefinition assembly)
+		{
+			switch (_context.Annotations.GetAction (assembly))
```

Comment:
@radekdoulik How do you want me to get to Annotations once I move this method to Tracer?